tools/cli/helpers/doc-parser/src/class-doc-parser.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Package description here
+ *
+ * @package automattic/jetpack-doc-parser
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Converts PHPDoc markup into a template ready for import to a WordPress blog.
+ */
+class Doc_Parser {
+
+	const PACKAGE_VERSION = '0.1.0-alpha';
+
+	/**
+	 * Generate a JSON file containing the PHPDoc markup, and save to filesystem.
+	 *
+	 * @param Array $args this function takes a path as its argument,
+	 * as well as optionally an output file name.
+	 */
+	public function generate( $args ) {
+		list( $directories, $output_file ) = $args;
+
+		if ( empty( $output_file ) ) {
+			$output_file = 'phpdoc.json';
+		}
+
+		$json = array();
+		foreach ( $directories as $directory ) {
+			$directory = realpath( $directory );
+			echo PHP_EOL;
+
+			// Get data from the PHPDoc
+			$json = array(
+				...$json,
+				$this->get_phpdoc_data( $directory ),
+			);
```

Comment:
Hmm. Since we flagged `tools/` with Jetpack-Compat-82 in #34207, we should do the same for parallel-lint. #34550 should fix that.

OTOH, I still think this would be more idiomatic.
```suggestion
			$json[] = $this->get_phpdoc_data( $directory );
```

Reply:
You're right, they are equivalent, I'm going to merge your suggestion.